app/main.py

```diff
@@ -2,0 +1,22 @@
+from distutils.file_util import copy_file
+import os
+
+
+def move_file(command: str) -> None:
+    if (command[:3]) != "mv " or command.count(" ") != 2:
+        return
+    _, old_file, new_file = command.split(" ")
+
+    if new_file.count("/") != 0:
+        path = ""
+        for i in new_file.split("/"):
+            if "." not in i:
+                path += i + "/"
+                if not os.path.isdir(path):
```

Comment:
Consider optimizing the code by avoiding multiple calls to `os.path.isdir` within the loop. You can store the result in a variable if the path doesn't change within the loop iteration.